easybuild/easyconfigs/r/R/R-3.2.0-ictce-7.3.5-bare.eb

```diff
@@ -0,0 +1,27 @@
+name = 'R'
+version = '3.2.0'
+versionsuffix = '-bare'
+
+homepage = 'http://www.r-project.org/'
+description = """R is a free software environment for statistical computing and graphics."""
+
+toolchain = {'name': 'ictce', 'version': '7.3.5'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = ['http://cran.us.r-project.org/src/base/R-%(version_major)s']
+
+preconfigopts = 'BLAS_LIBS="$LIBBLAS" LAPACK_LIBS="$LIBLAPACK"'
+configopts = "--with-lapack --with-blas --with-pic --enable-threads --with-x=no --enable-R-shlib"
+configopts += " --with-recommended-packages=no"
+
+dependencies = [
+    ('libreadline', '6.3'),
+    ('ncurses', '5.9'),
+    ('libpng', '1.6.17'),  # for plotting in R
+    ('libjpeg-turbo', '1.4.0'),  # for plottting in R
+    ('Java', '1.7.0_80', '', True),  # Java bindings are built if Java is found, might as well provide it
```

Comment:
and, ideally, the dep versions should be synced up with #1573 (in one direction or another)